_posts/2022-01-27-runway-to-sustainability-czi-funding.md

```diff
@@ -0,0 +1,48 @@
+
+In addition to paying mentors, call facilitators, experts, and directors, we intend to use this time to develop a sustainability plan beyond the end of 2023 - not only financial, but community sustainability. Our community is the centre of OLS, and we want to figure out ways to keep it healthy and thriving, serving its members and furthering our mission of open, equitable, and community-driven research.
+
+While we’ve been a registered company for some time now, we’re having to do a lot of growing up, too: establishing a payroll, learning about international tax and employment laws, meeting with accountants and writing policies.
+
+- **Stepping up: an Executive Director**: We are extremely delighted to announce that Yo Yehudi will join OLS as Executive Director starting from February 1st, to help drive the initiatives in this grant. The remaining members of the OLS Core Team (Emmy, Bérénice, Malvika) will also be recompensed to work on various components of OLS in different capacities.
+
+- **Establishing collaborations**: A core part of our sustainability planning will be working with and embracing other organisations interested in open research - organisations that might benefit from open research training and mentoring, including tailored inhouse deep-dives. It’s also essential to us that our core experience of OLS remain free and accessible to all, and that we retain focus on qualitative interactions, equity, and a global network.
+
+**How can you be part of this work?**
+
+## We’d love to work with you over the next 2 years.
+- Open-source research tools developers, maintainers and users- let’s co-design a module and curriculum!
+- **Governance planning and establishment**: As mentioned in our previous blog post, we aspire to be radically transparent in our governance. With you, we want to build and strengthen diverse pathways to community leadership, to ensure accountability and capacity to continue to serve our and other open communities.
+- **Collaborations**: Are you working in research or a related domain, and interested in collaborating, forming institutional partnerships, or hiring our services?
+
+For all of these - please do contact team@openlifesci.org to set up a chat!
+
+## In conclusion:
+
+We’re delighted to be able to direct paid staff time to looking forwards, rather than merely existing in the day-to-day to run the program, and we wouldn’t have been able to do this without the generous grant from CZI. Huge thanks in particular to Carly Strasser, Dario Taraborelli, and everyone on the CZI Open Science team who recognise the importance of both training and mentorship in open research practices, as well as providing well-needed financial stability to initiatives that previously have primarily been volunteer efforts. And as always - thank you to our community of mentors, experts, project leads, co-facilitators, partners and friends, who have supported and shaped OLS, tolerated our blunders and sustained our growth.
```

Comment:
```suggestion
We’re delighted to be able to direct paid staff time to look forwards, rather than merely existing in the day-to-day to run the program, and we wouldn’t have been able to do this without the generous grant from CZI. Huge thanks in particular to Carly Strasser, Dario Taraborelli, and everyone on the CZI Open Science team who recognise the importance of both training and mentorship in open research practices, as well as providing well-needed financial stability to initiatives that previously have primarily been volunteer efforts. And as always - thank you to our community of mentors, experts, project leads, co-facilitators, partners and friends, who have supported and shaped OLS, tolerated our blunders and sustained our growth.
```